src/main/java/views/main_menu/MainMenuV.java

```diff
@@ -1,35 +1,158 @@
  * to create a new month, and also allows the user to open the month menu for any of the
  * existing months in the current session
  */
-public class MainMenuV extends JPanel implements SessionLoadMainMenuVB {
-    public MainMenuV() {
+public class MainMenuV extends JPanel implements SessionLoadMainMenuVB, ActionListener {
+    private final MainMenuC controller;
+    private SessionStorage session;
+    private int selectedMonthID;
+    private final JComboBox<String> selectMonthComboBox;
+    private final JButton selectMonthButton;
+    private final JButton createMonthButton;
+    private final JButton saveButton;
+
+
+    public MainMenuV(MainMenuC controller) {
+        this.controller = controller;
+
         JLabel title = new JLabel("Main Menu");
         title.setAlignmentX(Component.CENTER_ALIGNMENT);
+        title.setFont(new Font(null, Font.PLAIN, 20));
 
-        this.setLayout(new BoxLayout(this, BoxLayout.Y_AXIS));
+        this.selectMonthComboBox = new JComboBox<>();
+        this.selectMonthButton = new JButton("Open month");
+        this.createMonthButton = new JButton("Create a new month");
+        this.saveButton = new JButton("Save");
+
+        // Disable dropdown and select button by default
+        // They will be enabled later in openMainMenu if there are any months to select
+        this.selectMonthComboBox.setEnabled(false);
+        this.selectMonthButton.setEnabled(false);
+
+        this.selectMonthComboBox.addActionListener(this);
+        this.selectMonthButton.addActionListener(this);
+        this.createMonthButton.addActionListener(this);
+        this.saveButton.addActionListener(this);
+
+        // Pack the dropdown and select buttons together in a JPanel
+        JPanel monthSelection = new JPanel();
+        monthSelection.setLayout(new BoxLayout(monthSelection, BoxLayout.X_AXIS));
+        monthSelection.add(this.selectMonthComboBox);
+        monthSelection.add(this.selectMonthButton);
+        monthSelection.setMaximumSize(new Dimension(300, 25));
+
+        monthSelection.setAlignmentX(Component.CENTER_ALIGNMENT);
+        monthSelection.setAlignmentY(Component.CENTER_ALIGNMENT);
+        this.saveButton.setAlignmentX(Component.CENTER_ALIGNMENT);
+        this.saveButton.setAlignmentY(Component.CENTER_ALIGNMENT);
+        this.createMonthButton.setAlignmentX(Component.CENTER_ALIGNMENT);
+        this.createMonthButton.setAlignmentY(Component.CENTER_ALIGNMENT);
+
+        this.add(new JPanel());
         this.add(title);
+        this.add(new JPanel());
+        this.add(monthSelection);
+        this.add(new JPanel());
+        this.add(this.createMonthButton);
+        this.add(new JPanel());
+        this.add(this.saveButton);
+        this.add(new JPanel());
+
+        this.setLayout(new BoxLayout(this, BoxLayout.Y_AXIS));
         this.setVisible(false);
     }
 
     /**
      * Makes the main menu visible which displays the session data passed and displays a message in a popup
```

Comment:
You could also rewrite this Javadoc to make it less repetitive (only say display once).

Reply:
Rewrote in new commit